apps/canvas/canvas.star

```diff
@@ -136,19 +163,22 @@ def fake_events():
     )
 
 def main(config):
-    api_token = config.get("msg", "")
+    api_token = config.get("api_token", "")
+    timezone = config.get("timezone", "America/Los_Angeles")
+    domain = config.get("domain", DEFAULT_DOMAIN)
```

Comment:
When a user is configuring the app, but hasn't entered a domain, I believe the empty string `""` will be passed as a value. Would be good to protect against that and use the default domain in that case as well.